test/function/array-intersects.yml

```
@@ -0,0 +1,125 @@
+name: "array::intersects() function"
```

```suggestion
name: "array::intersects() function"
version: ">= 1.2"
```

This needs a version for the test runners to actually run this test.